reflex/state.py

```diff
@@ -213,6 +213,18 @@ class BaseState(Base, ABC, extra=pydantic.Extra.allow):
     # The router data for the current page
     router: RouterData = RouterData()
 
+    # Custom exception hanlder
+    _exception_handler = None
+
+    @classmethod
+    def set_exception_handler(cls, exception_handler):
+        cls._exception_handler = exception_handler
+
+    @classmethod
+    def _handle_exception(cls, exception):
+        if cls._exception_handler:
```

Comment:
You should add a default exception handler which replicates the old behavior (just `print()` errors) and drop this check